csharp/ql/src/Likely Bugs/ThreadUnsafeICryptoTransformLambda.qhelp

```diff
@@ -3,8 +3,14 @@
   "qhelp.dtd">
 <qhelp>
 <overview>
-  <include src="ThreadUnsafeICryptoTransform.qhelp" />
-  
+<p>Classes that implement <code>System.Security.Cryptography.ICryptoTransform</code> are not thread safe.</p>
```

Comment:
You should be able to extract the shared blob of text using a fragment. See https://github.com/Semmle/ql/blob/22294091808d3af2418ecc3d53b8843eb274277d/java/ql/src/DeadCode/DeadMethod.qhelp#L10 for an example.

Reply:
Yeah, that is much nicer. Didn't know that was possible, thanks.

Reply:
I wasn't sure either until 15 minutes ago ;-)